database/migrations/2024_04_09_165945_update_people_table.php

```diff
@@ -38,6 +38,15 @@ public function up(): void
             $table->boolean('occupations_opt_out')->nullable();
             $table->integer('demographics_version')->nullable();
             $table->date('demographics_completed_date')->nullable();
```

Comment:
It looks like most of the other "timestamp"-like fields in the system use the dateTime type instead of date.